en/flight_modes/land.md

```diff
@@ -26,30 +26,132 @@ Landing is affected by the following parameters:
-<span id="COM_RC_OVERRIDE"></span>[COM_RC_OVERRIDE](../advanced_config/parameter_reference.md#COM_RC_OVERRIDE) | Controls whether stick movement on a multicopter (or VTOL in MC mode) causes a mode change to [Position mode](../flight_modes/position_mc.md). This can be separately enabled for auto modes and for offboard mode, and is enabled in auto modes by default.
-<span id="COM_RC_STICK_OV"></span>[COM_RC_STICK_OV](../advanced_config/parameter_reference.md#COM_RC_STICK_OV) | The amount of stick movement that causes a transition to [Position mode](../flight_modes/position_mc.md) (if [COM_RC_OVERRIDE](#COM_RC_OVERRIDE) is enabled).
+<a id="MPC_LAND_SPEED"></a>[MPC_LAND_SPEED](../advanced_config/parameter_reference.md#MPC_LAND_SPEED) | The rate of descent during landing. This should be kept fairly low as the ground conditions are not known.
+<a id="COM_DISARM_LAND"></a>[COM_DISARM_LAND](../advanced_config/parameter_reference.md#COM_DISARM_LAND) | Time-out for auto disarm after landing, in seconds. If set to -1 the vehicle will not disarm on landing.
+<a id="COM_RC_OVERRIDE"></a>[COM_RC_OVERRIDE](../advanced_config/parameter_reference.md#COM_RC_OVERRIDE) | Controls whether stick movement on a multicopter (or VTOL in MC mode) causes a mode change to [Position mode](../flight_modes/position_mc.md). This can be separately enabled for auto modes and for offboard mode, and is enabled in auto modes by default.
+<a id="COM_RC_STICK_OV"></a>[COM_RC_STICK_OV](../advanced_config/parameter_reference.md#COM_RC_STICK_OV) | The amount of stick movement that causes a transition to [Position mode](../flight_modes/position_mc.md) (if [COM_RC_OVERRIDE](#COM_RC_OVERRIDE) is enabled).
 
 
-## Fixed Wing (FW)
+## Fixed-wing (FW)
 
-The vehicle will turn and land at the location at which the mode was engaged.
-RC stick movement is ignored.
+Fixed-wing landing via mav command is currently not supported.
+The landing mode is only accessible in missions.
+Please see QGroundControl docs for more information on [planning a fixed-wing landing pattern mission item](https://docs.qgroundcontrol.com/master/en/PlanView/pattern_fixed_wing_landing.html).
 
-Fixed wing landing logic and parameters are explained in the topic: [Landing (Fixed Wing)](../flying/fixed_wing_landing.md).
+### Landing Sequence
+
+The fixed-wing landing pattern has three components: a **landing descent orbit**, a **landing waypoint**, and a **landing approach slope**.
+
+On engaging a landing pattern, the following sequence is initiated:
+
+1. **Fly to landing location**: The aircraft flies at the current altitude to the landing descent orbit.
+2. **Descend to approach altitude**: On reaching the orbit, the vehicle will descend to the approach altitude.
+   Once reaching the approach altitude, the airframe will continue on the orbit until reaching the final approach exit (tangential exit towards landing waypoint).

```

Comment:
I think what happens is that when landing is engaged.
1. You fly at current altitude until you enter some radius "landing descent orbit".
   - I think this is the "loiter point radius" in the landing pattern - right? 
   - What is this in MAVLink - i.e. what item and param?
2. You then do a "descending orbit" to decrease your altitude to the approach altitude.
   - Is the approach altitude the "loiter point altitude" in the landing pattern? Or is it something else.
4. You then continue orbit descent until you get to the final approach - i.e. there is a tangential path to the landing waypoint. Now you're in final approach you descent along that path with some settings.
   - I think the final approach altitude might be calculated based on `FW_LND_ANGLE` - i.e. you orbit until you have a good tangent, and if the land angle is less than the max defined by  `FW_LND_ANGLE` you exit the loiter and proceed. Is that right?
5. Then you flare for final landing.

My concern here is that it might be hard to plan the landing if we don't make it clear at what point you decide to leave each stage/what altitude or conditions move between the stages. I might be being thick.

Reply:
OK, so reading the docs I once wrote and should have remembered: https://docs.qgroundcontrol.com/master/en/PlanView/pattern_fixed_wing_landing.html#implementation

>This pattern creates three mission items:
> - [MAV_CMD_DO_LAND_START](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_LAND_START) - If you abort a landing it sends DO_GO_AROUND to the vehicle, which then causes the mission to return to this point and try to land again.
> - [MAV_CMD_NAV_LOITER_TO_ALT](https://mavlink.io/en/messages/common.html#MAV_CMD_NAV_LOITER_TO_ALT) - Start point for landing
> -[MAV_CMD_NAV_LAND](https://mavlink.io/en/messages/common.html#MAV_CMD_NAV_LAND) - End point for landing
> 
> The vehicle flares to landing using a flight path generated by the firmware between the NAV_LOITER_TO_ALT point and the NAV_LAND point.
>
> If those two locations violate the vehicle's flare constraints (e.g. descent angle is too steep) an error will be raised after you upload the invalid mission to the vehicle.

So based on the above (is it correct to your knowledge) and your comments:

I think what happens is that ...
1. Flying along in a mission, you see `MAV_CMD_DO_LAND_START`. 
   - From the comments in that command, in a mission this is just a placeholder so the system knows it is entering an FW landing sequence.
   - So that would immediately be accepted, ignoring its lat,lon,alt.
3. You then get a `MAV_CMD_NAV_LOITER_TO_ALT` which has a lat, lon, alt, radius
   - You fly towards this lat, lon 
   - When you reach the radius defined in item you do a loitering descent to MAV_CMD_NAV_LOITER_TO_ALT.alt = approach altitude.
4. YOu then continue to loiter down to **some** height where you exit the orbit (when vehicle has tangential path to the position (lat/lon/alt) in MAV_CMD_NAV_LAND)
   - The QGC docs kind of imply that after the loiter alt you have a path to flare that is up to the flight stack - which we have.
   - But it isn't clear how the vehicle decides it will exit the loiter
   - It isn't clear how the flare height is set?
   - Its a bit odd, because you're loitering when no longer in the loiter-to-alt - your're now in the NAV_LAND - right?
6. Then you flare for final landing at MAV_CMD_NAV_LAND

Can you confirm the behaviour and how flare height is set. If so, I think I get this :-)

Reply:
>Flying along in a mission, you see MAV_CMD_DO_LAND_START. 

DO_LAND_START is actually only used in RTL to find the land sequence. Not needed in Mission for the landing, there it just follows the WPs.

>2

Correct.

>YOu then continue to loiter down to some height

No, to **exactly** the approach altitude (altitude field of LOITER_TO_ALT). Once there you keep the altitude until the current course is aligned to the next waypoint, then exit. 

>YOu then continue to loiter down to some height

>It isn't clear how the flare height is set?

That's how it is formulated now, which parts would you want to have reformulated?
The flaring altitude is calculated during the final approach via "time-to-impact" ([FW_LND_FL_TIME](#FW_LND_FL_TIME)) and the approach descent rate. An additional safety parameter [FW_LND_FLALT](#FW_LND_FLALT) sets the minimum altitude at which the vehicle will flare (if the time based altitude is too low to allow a safe flare maneuver).

>Its a bit odd, because you're loitering when no longer in the loiter-to-alt - your're now in the NAV_LAND - right?

No you're no longer loitering then.... :thinking: 

>Then you flare for final landing at MAV_CMD_NAV_LAND

You flare once below the the flaring conditions as shown above, otherwise you simply keep descending at the constant descent speed along the final approach slope until ground contact. NAV_LAND is not a WP that has to be accepted in the normal sense- you need it to calculate your final approach (direction and gradient of slope).

Reply:
OK, so at the MAV_CMD_NAV_LOITER_TO_ALT.alt you loiter until are aligned to next waypoint. Then you head towards MAV_CMD_NAV_LAND. You follow a glide slope down, then flare at end. 

Got it now. Will tidy up tomorrow based on this.